enzoic/enzoic.py

```diff
@@ -71,6 +71,59 @@ def check_password(self, password: str) -> bool:
         else:
             return False
 
+    def check_hashed_password(self, hashed_pw: str, hash_type: int) -> bool:
+        """
+        Checks whether the provided password is in the Enzoic database of known, compromised passwords. Pass in the type
+        of password hash. Supports NTLM, MD5, SHA1, SHA256 (33, 1, 2, 3 respectively). You can utilize the PasswordTypes
+        enum for ease of use like so:
+
+        from enzoic.enums.password_types import PasswordType
+        PasswordType.NTLM
+
+        See: https://www.enzoic.com/docs/passwords-api
+        :param password: The full hash of a password you wish to check. Must match the corresponding hash_type
```

Comment:
I think this should be updated to hashed_pw and hash_type and we should include that it should be a partial hash containing the first 7 characters of the hashed password they wish to check.